src/cpu/x64/jit_brgemm_conv_bwd_strided.cpp

```diff
@@ -1513,6 +1515,7 @@ void brgemm_convolution_bwd_strided_t<isa>::ker_trans(
          is_first_call_postops_state_changed = false;
     const auto call_brgemm = [&](int brg_idx, int oc_block_s, int n_oc_blocks,
                                      size_t comp_ker_offs, bool do_postops) {
+        assert(brg_idx >= 0);
```

Comment:
Again, `assert` doesn't fix anything.
This lambda should start returning status. If `brg_idx` is negative, it should report `status::runtime_error`.
And it should start checking the status of `call_brgemm_kernel` if it has one.

Reply:
Updated. Now it's aligned with the same lambda in brgemm: https://github.com/oneapi-src/oneDNN/blob/main/src/cpu/x64/jit_brgemm_conv.cpp#L2048